app/services/paypal_service.rb

```diff
@@ -22,6 +37,92 @@ def self.execute_agreement(agreement_token)
 
   private
   
+  def activate_plan_params
+    {
+      op: 'replace',
+      value: { state: 'ACTIVE' },
+      path: '/'
+    }
+  end
+
+  def plan_params_with_trial
+    {
```

Comment:
one possibility to get round the codeclimate issues here could be to get these JSON things into erb templates? but maybe that should be a refactoring ticket after we have this out?

Reply:
@mattwr18 @tansaku I am not certain I understand how to implement this suggestion. do we create the erb under views/subscription directory? How do I pass the instance variables from service to the erb? maybe its a lil vague for me.

Reply:
does this help? https://www.stuartellis.name/articles/erb/

the idea is that you have a template called my_json.erb and you evaluate it - same as with erb on other places you set instance variables ...